### PR TITLE
[UID2-3515] CTV Integration Overview doc

### PR DESCRIPTION
**Pull request by genwhittTTD**
draft of CTV Integration Overview doc for AQ.
Note the file appears twice. Please do not review the second which is just a copy for the Japanese file set.
Not yet done: we need to add it in to the Publisher Overview, Integration Guides Summary, and possibly other docs. Not sure what to say exactly since it's an overview, not a guide.

### CODE REVIEW

File: docs/guides/integration-ctv-guide.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+| Integration Option | Token Is Generated... | Token Is Refreshed... |
+| :--- | :--- | :--- |
+| [Client-Side Integration](#client-side-integration-for-ctv-apps) | On the client side | On the client side |
+| [Server-Side Integration](#server-side-integration-for-ctv-apps) | On the server side | On the server side |
+| [Client-Server Integration](#client-server-integration-for-ctv-apps) | On the server side | On the client side |
+
+## Client-Side Integration for CTV Apps
+
+The client-side option is for publishers who want to perform both activities on the client side: generate the UID2 token, and then refresh the UID2 token as needed, in the CTV application.
```
Comment (sunnywu) [inline docs/guides/integration-ctv-guide.md:40]: why _application_? i thought we standardize into _app_?

File: docs/guides/integration-ctv-guide.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+This setup requires that all code changes are done on the client side.
```
Comment (sunnywu) [inline docs/guides/integration-ctv-guide.md:42]: _This setup requires that all code changes are done within the CTV app._

I think mentioning "client side" adds no value - might as well state _app_ clearly.

File: docs/guides/integration-ctv-guide.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+:::note
+The UID2 iOS SDK is also compatible with [Apple tvOS](https://developer.apple.com/tvos/). Similarly, the UID2 Android SDK is compatible with [Android TV](https://www.android.com/tv/).
+:::
+
+## Server-Side Integration for CTV Apps
+
+The server-side option is for publishers who want to perform both activities on the server side: generate the UID2 token, and refresh as needed, on the server side.
+
+This setup requires that most of the code changes are done on the server side, with minimal changes in the CTV application.
```
Comment (sunnywu) [inline docs/guides/integration-ctv-guide.md:54]: app?

File: docs/guides/integration-ctv-guide.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+To implement using this approach, follow the instructions in [Publisher Integration Guide, Server-Side](custom-publisher-integration.md).
+
+If your server-side code is in Java or Python, you can use one of the UID2 SDKs to make the HTTP requests to UID2, instead of writing your own source code. For details, refer to one of the following SDK guides:
+
+- [UID2 SDK for Java Reference Guide: Usage for Publishers](../sdks/uid2-sdk-ref-java.md#usage-for-publishers)
+- [UID2 SDK for Python Reference Guide: Usage for Publishers](../sdks/uid2-sdk-ref-python.md#usage-for-publishers)
+
+## Client-Server Integration for CTV Apps
+
+The client-server option is for publishers who want to generate UID2 tokens on the server side and then refresh them as needed from within the CTV application.
```
Comment (sunnywu) [inline docs/guides/integration-ctv-guide.md:65]: app?

File: docs/guides/integration-ctv-guide.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+:::note
+The UID2 iOS SDK is also compatible with [Apple tvOS](https://developer.apple.com/tvos/). Similarly, the UID2 Android SDK is compatible with [Android TV](https://www.android.com/tv/).
+:::
+
+## Server-Side Integration for CTV Apps
+
+The server-side option is for publishers who want to perform both activities on the server side: generate the UID2 token, and refresh as needed, on the server side.
+
+This setup requires that most of the code changes are done on the server side, with minimal changes in the CTV application.
```
Comment (asloobq) [inline docs/guides/integration-ctv-guide.md:53]: @genwhittTTD Can you please add another sentence here
>Another advantage of this approach is when you deal with multiple platforms (Web / CTV / mobile) doing everything server side can reduce platform specific efforts.

File: docs/guides/integration-ctv-guide.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-ctv-guide.md:39]: Done all instances

File: docs/guides/integration-ctv-guide.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/integration-ctv-guide.md:41]: Done.

File: docs/guides/integration-ctv-guide.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/integration-ctv-guide.md:53]: Done.

File: docs/guides/integration-ctv-guide.md (hunk 5 above)
Comment (genwhittTTD) [reply docs/guides/integration-ctv-guide.md:53]: Done, like this:
Another advantage of this approach is that if you're dealing with multiple platforms (Web/CTV/mobile), doing everything on the server side can reduce platform-specific efforts.

File: docs/guides/integration-ctv-guide.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/guides/integration-ctv-guide.md:64]: Fixed all instances.

**Review by asloobq — APPROVED** [review]

**Review by sunnywu — APPROVED** [review]

**Comment by asloobq** [comment]
changed title so that PR gets linked to JIRA ticket

**Pull request merged**